cernopendata/modules/fixtures/data/docs/delphi-data-release-2024/delphi-data-release-2024.md

```diff
@@ -1,8 +1,41 @@
-The DELPHI Collaboration decided to open their data and released it to the physics community for public access, following FAIR principles. (FIXME: give more information here when ready.)
+The DELPHI Collaboration decided to open their data and released it to the physics community for public access, following FAIR principles.
+DELPHI was one of four detectors at the LEP collider, which collided electrons and positrons, at until now unreached energies. Since the two
+particles annihilate entirely into pure energy, the total energy of the events is precisely known, and the end states are much cleaner than
+this is the case for hadron colliders like LHC.
 
-<p><center><img src="/static/docs/delphi-data-release-2024/t302035_001_r109522_e013843.png" width="60%"></center></p>
+
+Along with the data, the collaboration decided to open
+the original software stack, including simulation and reconstruction. This enables scientists to pass the output of decent generators through the detector simulation of the DELPHI detector,
+and compare to what was actually measured at the time. It is also possible to use this data for educational purpose. The available data and software allow to either pick and reprocess individual
+events from raw data, or create simulated events of a specific end state, pass them through the detector response, and then scan them using the event display.
+
+
+The publication includes:
+
+DELPHI Data sets:
+
+* The complete [raw data](/search?q=&f=experiment%3ADELPHI&f=file_type%3ARAWD&l=list&order=desc&p=1&s=10&sort=mostrecent) set, for the years 1990-2000, about 6 Tb
```

Comment:
Cosmetics: 6 TB (units in all uppercase)

Reply:
Fixed during rebase.

Reply:
looks good! Thanks!